include/taskolib/SequenceManager.h

```diff
@@ -219,6 +224,17 @@ class SequenceManager
      */
     static UniqueId create_unique_id(const std::vector<SequenceOnDisk>& sequences);
 
+    /**
+     * Stage all files git can find in a repository.
+     * Two globs can be specified to find only the files that are wanted.
+     *
+     * \param glob1  Main pathname specification which files to add as glob.
+     *               If it is "" all files in the repo are considered.
+     * \param glob2  Secondary pathname specification. If it is "" it is ignored.
+     * \return       Commit message starting with a linebreak.
```

Comment:
Reading this, I do not really understand why we have two globs. Maybe we could add an example or a few more words of explanation?

Reply:
Thanks for bringing this up. At 4 am it looked like the natural thing to do, because both globs are handled differently.
But in the light of the day it looks ridiculous. Changed.